app/components/goodcity/organisation-search-overlay.js

```diff
@@ -1,10 +1,13 @@
 import Ember from "ember";
+import _ from "lodash";
+import SearchMixin from "stock/mixins/search_resource";
 
-export default Ember.Controller.extend({
+export default Ember.Component.extend(SearchMixin, {
```

Comment:
I think you pasted `Component` by mistake here : )

Reply:
no 🤔 
This should be `Component` as its an overlay component 🤔

Reply:
Oh, I didn't notice you moved it to another folder ! Nice one !
I saw `app/controllers` so thought it was weird